AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingViewController.swift

```diff
@@ -371,6 +400,7 @@ class MeetingViewController: UIViewController {
     @IBAction func leaveButtonClicked(_: UIButton) {
         meetingModel?.endMeeting()
         deregisterFromKeyboardNotifications()
+        deregisterForAppLifeCycleNotification()
```

Comment:
Will these be called if we leave meeting from CallKit?

Reply:
See the tests on the description